
GRAVE HACK. I need to review what the assemble "is thinking".

  - Rene Rebe <rene@exactcode.de>

--- linux-2.4.27/arch/arm/Makefile.vanilla	2005-02-02 00:12:26.648839824 +0100
+++ linux-2.4.27/arch/arm/Makefile	2005-02-02 00:12:09.753408320 +0100
@@ -27,7 +27,7 @@
 #
 # Note!  For APCS-26 YOU MUST HAVE AN APCS-26 LIBGCC.A
 #
-apcs-y				:=-mapcs-32
+apcs-y				:=-mapcs-32 -mcpu=arm710t -Wa,-mcpu=arm710t
 apcs-$(CONFIG_CPU_26)		:=-mapcs-26 -mcpu=arm3 -Os
 
